ashpd-demo/src/main.rs

```diff
@@ -24,7 +24,12 @@ fn main() -> glib::ExitCode {
 
     gst4gtk::plugin_register_static().expect("Failed to register gstgtk4 plugin");
 
-    let res = gio::Resource::load(RESOURCES_FILE).expect("Could not load gresource file");
+    let argv0 = std::env::args().next();
+    let res = if argv0.is_some() && argv0.unwrap().ends_with(".devel") {
+        gio::Resource::load(BUILDDIR_RESOURCES_FILE)
+    } else {
+        gio::Resource::load(RESOURCES_FILE)
+    }.expect("Could not load gresource file");
```

Comment:
I am not particularly happy with this approach of checking the binary name. I think a viable path forward is to have a separate meson build option that would influence from where we load the gresource file making the changes to config.rs/main.rs not needed at all. The other option is to embed the gresource into the final binary but nobody took the time to figure out a nice way to do it that integrates well with the hackish meson/cargo thing we are doing